lib/cocoapods-search/command/search.rb

```diff
@@ -82,7 +84,19 @@ def local_search
           sets.reject! { |set| !set.specification.available_platforms.map(&:name).include?(platform) }
         end
 
-        sets.reverse_each do |set|
+        if @use_pager
+          IO.popen((ENV['PAGER'] || 'less -R'), 'w') do |io|
```

Comment:
Actually I wanted to do as you suggest, but I have to call any method doing printing inside this popen block. Therefore I couldn't find a proper way to extract this part, and call `print_sets` only once. 

Maybe I can make `with_pager` taking a block parameter, and set that block as the block of popen if needed. 

Do you have a better suggestion?